autoPyTorch/utils/implementations.py

```diff
@@ -35,7 +35,7 @@ def __call__(self, y: Union[np.ndarray, torch.Tensor]) -> np.ndarray:
             weights = (np.ones(y.shape[1]) * weight_per_class) / np.maximum(counts, 1)
         else:
             classes, counts = np.unique(y, axis=0, return_counts=True)
```

Comment:
Just to make sure, The index of the weights should be correspondence to the class index, right?